### PR TITLE
Adds artifical muscle tissue rig module

### PR DESCRIPTION
**Pull request by jknpj**
I know, this isn't really balanced or well designed but it's a bus module(for now) and i just wanted to make a future PR less unatomic.

### CODE REVIEW

File: code/modules/clothing/spacesuits/rig_modules/rig_modules.dm [hunk 1]
```diff
@@ -119,4 +119,31 @@
...
\ No newline at end of file
+		R.H.clothing_flags &= ~PLASMAGUARD
+
+/obj/item/rig_module/muscle_tissue
+	name = "artificial muscle tissue"
+	desc = "A flexible tissue with a number of sensors stretched between its surface and interior of the suit. When these sensors detected an impact, the artificial muscle reacts instantaneously, contracting and diffusing the damage."
+
+/obj/item/rig_module/muscle_tissue/activate(var/mob/user,var/obj/item/clothing/suit/space/rig/R)
+	..()
+	if(R.cell.use(5000))
```
Comment (Kurfursten) [inline code/modules/clothing/spacesuits/rig_modules/rig_modules.dm]: I've always thought the way the rig module system works is weird

Turning your hulk on and off 10 times costs 50,000 charge but leaving hulk on all shift costs 5,000

Comment (jknpj) [reply code/modules/clothing/spacesuits/rig_modules/rig_modules.dm]: Maybe a processing object is better? It's kinda weird how we cannot disable rig modules individually, too.

Comment (Kurfursten) [reply code/modules/clothing/spacesuits/rig_modules/rig_modules.dm]: Yes, I would definitely prefer them as processing

**Comment by Caboozles** [comment]
So while Hulk is pretty broken, is there any way a version of this could be made where you get a minor variant that lets you be able to pick things up like strong? 

I mostly just want this for the memes of being able to reliably throw barrels. Or anvils.

**Comment by jknpj** [comment]
Weirdly enough, this alone doesn't do much, because the pain damage from tasers bolts will still fuck you up.

**Comment by Kurfursten** [comment]
✅ I like this process system better and hope the rest of our rig stuff gets adapted for it. Maybe it should be a module level var for passive energy use and be in the module-level process() to handle using energy if activated and deactivate if the battery runs out.

**Comment by Blithering** [comment]
>pain damage from taser bolts will still fuck you up

Is there a way to have it give you toggle-able oxycodone effect? "Doktor, turn off my pain inhibitors!"

**Comment by jknpj** [comment]
@Kurfursten i'll see if i can mess with that later on

**Comment by jknpj** [comment]
> 
> 
> > pain damage from taser bolts will still fuck you up
> 
> Is there a way to have it give you toggle-able oxycodone effect? "Doktor, turn off my pain inhibitors!"

Maybe another rig_module just for that?

**Pull request merged**